seeds/01_consent.ts

```diff
@@ -95,6 +96,7 @@ const consents = [
   }
 ]
 
-export function seed (knex: Knex): Knex.QueryBuilder<number[]> {
-  return knex('Consent').insert(consents)
+export function seed (knex: Knex): Promise<Knex.QueryBuilder<number[]>> {
+  return knex('Consent').del()
```

Comment:
do we really want to delete all entries? please add a comment on why.

Reply:
from [the knex documentation](http://knexjs.org/#Seeds-CLI): "You should design your seed files to reset tables as needed before inserting data."

Our seed files were not resetting the table as needed so when we would seed a table more than once, there would be duplicate entry errors etc.

Reply:
So we want to clear all the entries before populating them again with the entries defined in the seed file so that we are in a good state.